### ISSUE TITLE
Web Build Broken

### ISSUE THREAD

**Issue by abueide** [joreilly/PeopleInSpace#70]
web:jsBrowserRun task fails with

```
[webpack-cli] Invalid configuration object. Object has been initialized using a configuration object that does not match the API schema.
   - configuration has an unknown property '_assetEmittingWrittenFiles'. These properties are valid:
     object { bonjour?, client?, compress?, dev?, firewall?, headers?, historyApiFallback?, host?, hot?, http2?, https?, liveReload?, onAfterSetupMiddleware?, onBeforeSetupMiddleware?, onListening?, open?, port?, proxy?, public?, setupExitSignals?, static?, transportMode?, watchFiles? }
```

**Comment by abueide** [comment]
https://youtrack.jetbrains.com/issue/KT-48273

**Comment by abueide** [comment]
@joreilly this broke all kotlinJs projects using webpack retroactively even if you didn't change anything in your codebase, so I don't think you missed anything

**Comment by hfhbd** [comment]
This fix is included in Kotlin 1.5.30, so just update Kotlin and Compose to build328.
